channels/1.25

```diff
@@ -0,0 +1,23 @@
+manifests:
+- name: api-manager
+  version: 1.2.3
+- name: csi
+  version: 0.1.0
+- name: node
+  version: 2.5.0
```

Comment:
The `2.5.0` here is really just the package name for the manifests, actual image versions etc are overwritten.

Reply:
For instance, it just points to this package https://github.com/storageos/operator/tree/main/channels/packages/node/2.5.0
the node manifests haven't had any API updates or changes since 2.5.0 so it's the only package we have for node.

Reply:
Ah ok, I always forget how these channels work 😅